frontend/talentsearch/src/js/components/Router.tsx

```diff
@@ -46,6 +47,12 @@ const routes = (
       component: <ProfilePageApi />,
     }),
   },
+  {
+    path: profilePaths.governmentInformation(),
+    action: () => ({
+      component: <GovInfoFormContainer />,
+    }),
+  },
```

Comment:
resolved or outstanding?

Suppose this could be what gives this closure to the open question, yes? https://github.com/GCTC-NTGC/gc-digital-talent/issues/2363

Reply:
We can ignore the issue in this PR, and fix it after closing #2363.